### PR TITLE
uCredit Advertisements

### PR DESCRIPTION
**Pull request by yesterd3v**
Please review the [guidelines for contributing](https://github.com/jhuopensource/semesterly/blob/master/CODE_OF_CONDUCT.md) to this repository.
### Description - Describe your pull request
* Introduce uCredit + Semester.ly partnership

### If applicable, add a checklist of incremental tasks for this PR.
- [x] Create button to link to uCredit
- [x] Temporary Patch for press page (bug with one of the images) -> #614 

### Your Checklist
- [x] Linked Issue and/or Project?
- [x] Added Reviewer, Assignee, and/or Label?

### CODE REVIEW

**Comment by yesterd3v** [comment]
@JJamesWWang In regards to the design for the front page, I think it'd be clearer to have a button, add a horizontal line to separate the section, possibly also reduce the height, and fix the text alignment. Here's a general idea ->
Screen Shot 2021-10-12 at 10 53 31 AM


I have this bug when I opened it locally:
Screen Shot 2021-10-12 at 10 51 49 AM

File: build/local_settings.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import os
 
-DEBUG = True
+DEBUG = False
```
Comment (JJamesWWang) [inline build/local_settings.py]: Keep these true

**Review by JJamesWWang — CHANGES_REQUESTED** [review]
Can you also screenshot it so I don't have to rebuild my container

File: semesterly/settings.py [hunk 2]
```diff
@@ -60,7 +60,7 @@ def get_secret(key):
 
 SECRET_KEY = get_secret("SECRET_KEY")
 
-DEBUG = True
+DEBUG = False
```
Comment (JJamesWWang) [inline semesterly/settings.py]: And this one

File: build/local_settings.py (hunk 1 above)
Comment (yesterd3v) [reply build/local_settings.py]: I didn't change these, I think you did, but I'll fix it now

Comment (yesterd3v) [reply build/local_settings.py]: I'll screenshot, but I don't think you have to rebuild the container?

**Comment by yesterd3v** [comment]
> Can you also screenshot it so I don't have to rebuild my container

New section of front page. The button links to [ucredit.me](https://ucredit.me/).
Screen Shot 2022-01-25 at 11 48 00 AM

**Review by JJamesWWang — APPROVED** [review]
Looks good

**Pull request merged**